dart/test/sentry_test.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:mockito/mockito.dart';
+import 'package:sentry/src/client.dart';
+import 'package:sentry/src/protocol.dart';
+import 'package:sentry/src/sentry.dart';
+import 'package:sentry/src/sentry_options.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Sentry.init', () {
+    SentryClient client;
+
+    Exception anException;
+
+    final dns =
+        'https://cb0fad6f5d4e42ebb9c956cb0463edc9@o447951.ingest.sentry.io/5428562';
+
+    setUp(() {
+      Sentry.init(SentryOptions(dsn: dns));
+
+      client = MockSentryClient();
+      Sentry.initClient(client);
+    });
+
+    test('should capture the event', () {
+      Sentry.captureEvent(event);
+      verify(client.captureEvent(event: event)).called(1);
+    });
+
+    test('should capture the exception', () {
+      Sentry.captureException(anException);
+      verify(client.captureException(exception: anException)).called(1);
+    });
+  });
+}
+
+class MockSentryClient extends Mock implements SentryClient {}
+
+final event = Event(
```

Comment:
not possible to reuse the `event_example.dart ` right? just curious if there's some magic to share code across tests/samples

Reply:
not really as the test files are not exposed in the library.
We could set a relative path in the test file `import '../example/event_example.dart';` but I'm not sure I'll vote for this, do you ?

Reply:
nah, not really